src/main/java/com/WebController/EventController.java

```diff
@@ -33,9 +33,12 @@
         PatientEventAssociation association;
         boolean verified =false;
         if (principal!=null){ // check if its a verified user
-            verified = this.userRepo.findUserByEmailAddressIgnoreCase(principal.getName()).isVerified();
+            verified = true;
         }
         for (Event e: events){
+            if (e.getDate()==null || e.getName().equals("") || e.getPlace().getAddress().equals("") || e.getPlace().getLongitude()==0.0 || e.getPlace().getLatitude()==0.0){
```

Comment:
I am not sure why you needed to add this.

Reply:
I am worried that users may enter some garbage data. For example an event without event name or without location data etc.